<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fileSelector</class>
 <widget class="QMainWindow" name="fileSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>File selector</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="newFileBtn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>550</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>New File</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>751</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="filesGrid" rowminimumheight="0">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Files:</string>
    </property>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New file</string>
   </property>
  </action>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionDelete_file">
   <property name="text">
    <string>Delete file</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
